cmd/kusk/cmd/mock.go

```diff
@@ -128,20 +79,20 @@ $ kusk mock -i https://url.to.api.com
 		if err != nil {
 			err := fmt.Errorf("unable to fetch user's home directory: %w", err)
 			reportError(err)
-			ui.Fail(err)
+			kuskui.PrintError(err.Error())
```

Comment:
Can you not just make this take an `error` object?

I see `kuskui.PrintError(err.Error())` being repeated in a lot of places.

Reply:
Print error accepts array of strings which is usually practical for constructing different error messages. So i'd leave it like this